nilearn/datasets/func.py

```diff
@@ -16,83 +16,10 @@
 from .._utils import check_niimg
 from .._utils.compat import BytesIO, _basestring, _urllib
 from .._utils.numpy_conversions import csv_to_array
-from .._utils.exceptions import VisibleDeprecationWarning
```

Comment:
Amusing case of cross-over between PRs: this removal cause https://github.com/nilearn/nilearn/pull/2308 to fail, after merging.

Addressed in: https://github.com/nilearn/nilearn/pull/2321

I think that with the version of numpy that we currently depend upon, we don't need to have a backport of VisibleDeprecationWarning anymore. Let's see what CI says on my other PR, and if it's happy I'll remove our backport.

Reply:
But serious question. Why do we use VisibleDeprecationWarning when Python's built-in FutureWarning does the same thing?

Reply:
Let's not get side-tracked and first worry about getting CI to pass on master.

Reply:
Thanks for fixing this !